inc/Ui/Ui.php

```diff
@@ -0,0 +1,20 @@
+<?php
+namespace dokuwiki\Ui;
+
+/**
+ * Class Ui
+ *
+ * Abstrract base class for all DokuWiki screens
```

Comment:
I just wanted a base for all the UI screens to be able to add base functionality later on (not sure what that functionality would be).